examples/typescript/README.md

```diff
@@ -0,0 +1,44 @@
+# saucectl puppeteer & typescript example
+
+Example running saucectl with puppeteer & typescript.
+
+## What You'll Need
+
+The steps below illustrate one of the quickest ways to get set up. If you'd like a more in-depth guide, please check out
+our [documentation](https://docs.saucelabs.com/testrunner-toolkit/installation).
+
+### Install `saucectl`
+
+```shell
+npm install -g saucectl
+```
+
+### Set Your Sauce Labs Credentials
+
+```shell
+saucectl configure
+```
+
+## Ensure you already have typescript installed
+
+```shell
+npm install -g typescript
```

Comment:
I don't get why this is needed before running `saucectl run` ?
Correct me if I'm wrong, you run the transpilation before calling `saucectl run` ?

Why not making all the typescript execution directly in the puppeteer runner ?

In my opinion, `one of the quickest ways to get set up` mean, I just have to clone the repo + execute `saucectl run`.

Reply:
You are right, first of all, we need to compile typescript to javascript, and only after that, we can run tests. I wanted to show all steps including compilation.